php/WP_Mock/Exceptions/WP_MockException.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace WP_Mock\Exceptions;
+
+use Exception;
+use Throwable;
+
+/**
+ * Base WP_Mock Exception.
+ */
+class WP_MockException extends Exception
```

Comment:
Just a question: why the `WP_Mock` prefix? It's already in a `WP_Mock` namespace, so it seems unnecessary. Why not something like `WP_Mock\Exceptions\BaseException` ?

Reply:
Good point, maybe I'll just rename that to `Exception` in the `WP_Mock` namespace; more specific exceptions can define better names